res/values-notnight-v21/styles.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
+
+<!--    <style name="SessionBottomSheetDialogTheme" parent="Theme.Design.Light.BottomSheetDialog">-->
```

Comment:
Are we still going to use this? If not let's remove it rather than just commenting it out :)

Reply:
Nicely spotted! I missed that one.